node-fs-writefile/cp.ts

```diff
@@ -0,0 +1,10 @@
+import { writeFile, readFile } from 'node:fs/promises';
+const target = process.argv[process.argv.length - 2];
+const destination = process.argv[process.argv.length - 1];
```

Comment:
can do something like this:

const [, , source, destination] = process.argv;

a bit simpler